src/main/java/com/monumental/services/AsyncMonumentService.java

```diff
@@ -0,0 +1,100 @@
+package com.monumental.services;
+
+import com.amazonaws.SdkClientException;
+import com.monumental.models.Image;
+import com.monumental.models.Monument;
+import com.monumental.repositories.MonumentRepository;
+import com.monumental.util.async.AsyncJob;
+import com.monumental.util.csvparsing.CsvMonumentConverterResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * This class is the async counterpart to the MonumentService, where any asynchronous tasks
+ * should be kept.
+ */
+@Service
+public class AsyncMonumentService extends AsyncService {
```

Comment:
@njd9419 I split this into a separate service for clarity, but it could just as easily work within the `MonumentService` if it extended `AsyncService`